src/content/docs/quick-tips/import-export.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: Import - export data
+description: Import - export data
+---
+
+Your data is yours to keep and manage. \
+Ontime provides features to allow you to manage your project files, including exporting your data to use somewhere else.
+
+## Export / Import Ontime project files
+
+### Exporting project file
+You can export and import project files from the project manager `Editor` -> `Settings` -> `Manage projects`.
+The project file is a `JSON` formatted file that includes your Rundown and all the application settings.
+
+### Importing project file
+If you are sharing project files across users or projects, you can import a previous `JSON` formatted file in the project manager `Editor` -> `Settings` -> `Manage projects`.
+
+Once a file is imported into Ontime, you can also use `Partial Load` to load only parts of another project into the currently loaded project. \
+This is a great way to create preset files that are reusable across your projects.
+
+## Export rundown data to consume elsewhere
+You may not be consuming Ontime's data in another Ontime instance. \
+In these cases, you would likely want to export just your Rundown since the other data is irrelevant outside the app.
+
+Ontime allows exporting the Rundown of any project to `CSV` format so that it can be consumed by other software and integrated with spreadsheets.
+
+Additionally, if a live connection is required, the [Google Sheet integration](/features/import-spreadsheet-gsheet) has the ability to read and write to a remote sheet.
```

Comment:
I think that `live` gives the indication that it happens automatically

Reply:
How about `persistent connection`?

Reply:
maybe
```suggestion
Additionally, you can upload/download directly from [Google Sheet integration](/features/import-spreadsheet-gsheet).
```

it's not to important